CMakeLists.txt

```diff
@@ -676,17 +681,6 @@ if(BUSTED_PRG)
   set(FUNCTIONALTEST_PREREQS nvim printenv-test printargs-test shell-test streams-test tty-test ${GENERATED_HELP_TAGS})
   set(BENCHMARK_PREREQS nvim tty-test)
 
-  # Useful for automated build systems, if they want to manually run the tests.
```

Comment:
good call. The more random things we have, the harder it is to make sense of the system, document it, and know if these little hidden features are even working :)